cupy/cuda/compiler.py

```diff
@@ -166,12 +164,8 @@ def _get_arch_for_options_for_nvrtc(arch=None):
     # generate cubin (SASS) instead of PTX. See #5097 for details.
     if arch is None:
         arch = _get_arch()
-    if driver._is_cuda_python():
-        version = runtime.runtimeGetVersion()
-    else:
-        version = _cuda_hip_version
     if (
-        not _use_ptx and version >= 11010
+        not _use_ptx
```

Comment:
FYI. I think this change is OK. For a more robust treatment for `_get_arch()`, I suggest to take a look at [this helper function](https://github.com/cupy/cupy/blob/1ef40363dbc0c489486a178f39b7b3b4ed223b7a/tests/cupy_tests/core_tests/test_raw.py#L551-L562) that I added recently. We don't have to do it in this PR or even in v13, since right now we almost always generate SASS which is problem free. But it is possible that the nvcc/nvrtc version is newer than the driver version, and the generated PTX would not be loadable. (I hit this when updating the PTX tests in `test_raw.py`, which prompted me to write that helper 🙂)